tests/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
 find_package(Catch2 3.1.0 REQUIRED)
+find_package(range-v3 0.11.0 REQUIRED)
```

Comment:
Do we really need version 0.11.0? This prevents this package to be build for Humble on Ubuntu Focal (which should be supported)
https://packages.ubuntu.com/source/focal/range-v3